package.json

```diff
@@ -14,12 +14,15 @@
     "execa": "4.0.0",
     "husky": "4.2.3",
     "lint-staged": "10.0.8",
+    "mocha": "7.1.0",
     "mri": "1.1.4",
+    "npm-run-all": "4.1.5",
```

Comment:
let's stay away from the npm-run-all, there is no real benefit.. and makes it harder when copying tasks directly to command line

Reply:
> no real benefit

I can see a benefit here

* the tasks tend to grow and grow and this composure with `yarn <task> & yarn <task> & yarn ...` will be clearly unreadable in some point
* npm-run-all gives us the chance to execute tasks in parallel in future (`run-p`)

Reply:
> the tasks tend to grow and grow and this composure with `yarn <task> & yarn <task> & yarn ...` will be clearly unreadable in some point

Yet another reminder to keep them to the minimum

> npm-run-all gives us the chance to execute tasks in parallel in future (`run-p`)

If we ever going to gain anything out of we can rethink.. I would rather have only one way to run tasks.. using `yarn`

Reply:
alrighty